internal/controller/datadogagent/utils.go

```diff
@@ -135,3 +140,32 @@ func getReplicas(currentReplicas, newReplicas *int32) *int32 {
 
 	return apiutils.NewInt32Pointer(*newReplicas)
 }
+
+// getDDAICRDFromConfig is only used in tests
```

Comment:
Ended up using this that gets the CRD from the `config` directory. Go embed wasn't working because of how the directory is structured